js/layout-javascript/news-feed-code.js

```diff
@@ -392,12 +386,13 @@ DynamicList.prototype.attachObservers = function() {
           _this.showDetails(entryId).then(function() {
             setTimeout(function() {
               _this.$closeButton.focus();
-              _this.$detailsContent.focusin(function() {
-                _this.focusCloseButton(false);
-              }).focusout(function() {
-                _this.focusCloseButton(true);
+              _this.$detailsContent.focusout(function(event) {
+                if (event.currentTarget.contains(event.relatedTarget)) {
+                  return;
+                }
+    
+                _this.$closeButton.focus();
               });
-            }, 200);
```

Comment:
@OleksandrLavrynenko Why would you remove this line?

Reply:
@OleksandrLavrynenko The removal of this line means code wouldn't have worked since it becomes syntactically incomplete. I see the code has gone past interview review and marked as ready for testing. Please test your own work before submitting for review or testing. cc @upplabs-alex-levchenko